src/main/java/io/cryostat/recordings/RecordingArchiveHelper.java

```diff
@@ -581,17 +580,13 @@ public boolean deleteReports(String subdirectoryName, String recordingName) {
                                     .resolve(subdirectoryName)
                                     .resolve(reportName)
                                     .toAbsolutePath();
-                    if (!fs.deleteIfExists(reportPath)) {
-                        logger.warn("Failed to delete report {}", reportPath);
-                        deleted = false;
-                    } else {
-                        logger.trace("Deleted report {}", reportName);
+                    if (fs.exists(reportPath)) {
+                        fs.deleteIfExists(reportPath);
                     }
                 }
             }
-            return deleted;
+            return true;
         } catch (IOException e) {
-            logger.warn(e);
```

Comment:
Any reasons behind removing warnings here and above?

Reply:
The exceptions will happen if there were never any reports generated in the first place, and thus the files aren't created. This would probably happen often in the case of the agent pushing, and the exception it logs is rather big. I can replace it with a smaller warning.

Reply:
Ahh make sense! Thanks for explaining :D